snapcraft.yaml

```diff
@@ -1393,6 +1393,15 @@ parts:
       go build -o "${CRAFT_PART_INSTALL}/bin/lxd-benchmark" github.com/canonical/lxd/lxd-benchmark
       go build -o "${CRAFT_PART_INSTALL}/bin/lxd-user" github.com/canonical/lxd/lxd-user
 
+      # Build the static website
+      apt install -y shellcheck python3.10-venv
```

Comment:
that this is the only occurrence of `apt` in this file should be an alarm to us that we have got this step wrong.

It looks like we need to use `build-packages` instead e.g  https://github.com/canonical/lxd-pkg-snap/pull/136/files#diff-e68b9afd67bb77dec8eda77796d3e0b784c7011203114996383eed6533be0cefL1340